src/maps/UserPreferences.php

```diff
@@ -63,6 +63,8 @@ class UserPreferences implements MapInterface
 
     private int $useMarkdown = 0;
 
+    private int $useOve = 1;
```

Comment:
I think it's fine if it's always working, no? There is no need to have a setting like this IMHO.

Reply:
There might be situations where a user want to switch the plasmid viewer off.
What if some file as an extension that matches any of the plasmid viewer extensions but it is actually something entirely different. Maybe in physics or engineering there are file formats that use the extension .dna, ..., or .gb but they don't contain what biologists / the plasmid viewer expects. The viewer will only show garbage or some errors. Maybe than it is better to switch it off?

Reply:
No I think in that case it should just display an error like "Invalid DNA data" and that's it. What if you also have valid dna files? It adds complexity for no good reasons IMHO.

Reply:
I see your point and it makes sense.